CMake/External_Darknet.cmake

```diff
@@ -0,0 +1,89 @@
+set(Darknet_DEPENDS)
+
+if(fletch_ENABLE_OpenCV)
```

Comment:
Fully agree with CUDA/CUDNN because of the mess of version complications. Less convinced with OpenCV, but willing to go along with it anyway. Just look for ways to keep the complexity of fletch options to a minimum. At least it is advanced and the default is on. Good to go.